lib/conjur/fpm/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build from the same version of ubuntu as phusion/baseimage
-FROM cyberark/phusion-ruby-fips:latest
+FROM registry.tld/cyberark/phusion-ruby-fips:0.11-9c1da20
```

Comment:
Should be changed to released version

Reply:
We should not be hardcoding the exact version here, and need to reference the dockerhub image (`cyberark/phusion-ruby-fips`, not `registry.tld/cyberark/phusion-ruby-fips`).

Reply:
Will rollback after [cyberark/conjur-base-image#71](https://github.com/cyberark/conjur-base-image/pull/71)  is pushed to master.
The push order is `conjur-base-image => debify => conjur-oss`

Reply:
Rolled back.